pkg/security/podsecuritypolicy/provider.go

```diff
@@ -310,6 +316,10 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		}
 	}
 
+	if !s.psp.Spec.AllowPrivilegeEscalation && sc.AllowPrivilegeEscalation != nil && *sc.AllowPrivilegeEscalation {
```

Comment:
the naming convention was taken from psp.Privileged which I also agree is confusing